code/game/data_huds.dm

```diff
@@ -302,6 +302,7 @@
 		simple_status_hud.icon_state = ""
 		infection_hud.icon_state = "robot"
 
+	var/is_bot = SEND_SIGNAL(src, COMSIG_HUD_NO_CLIENT)
```

Comment:
This isn't a signal it's a getter
You should make a proper flag to return instead of a bool

Reply:
done